### ISSUE TITLE
warning when creating VoltageDivider

### ISSUE THREAD

**Issue by peendebak** [QCoDeS/Qcodes#804]
Minimal example:
```
import qcodes
import  qcodes.instrument_drivers.devices
from qcodes.instrument_drivers.devices import VoltageDivider

p = qcodes.Parameter('p', set_cmd=None)
R = VoltageDivider(p, 4)
```
The warning emitted is 
```
c:\svn\qcodes\qcodes\instrument\parameter.py:207: UserWarning: Wrapping get method, original get method will not be directly accessible. It is recommended to define get_raw in your subclass instead.
```

**Comment by jenshnielsen** [comment]
This should be fixed by #802 now

**Comment by WilliamHPNielsen** [comment]
Since #802 is merged, I close this issue.
